ports/servoshell/egl/ohos.rs

```diff
@@ -447,6 +448,24 @@ pub fn register_url_callback(cb: JsFunction) -> napi_ohos::Result<()> {
     Ok(())
 }
 
+#[napi]
+pub fn register_prompt_toast_callback(cb: JsFunction) -> napi_ohos::Result<()> {
+    debug!("register_prompt_toast_callback called!");
+    let tsfn: ThreadsafeFunction<String, ErrorStrategy::Fatal> =
```

Comment:
Is the type annotation really necessary here?

```suggestion
    let function: ThreadsafeFunction<String, ErrorStrategy::Fatal> =
```

Reply:
Renamed and removed type annotation.